src/Microsoft.AspNet.Mvc.ModelBinding/ModelStateDictionary.cs

```diff
@@ -41,29 +58,50 @@ public ModelStateDictionary([NotNull] ModelStateDictionary dictionary)
         }
 
         /// <summary>
-        /// Gets or sets the maximum allowed errors in this instance of <see cref="ModelStateDictionary"/>.
-        /// Defaults to <see cref="int.MaxValue"/>.
+        /// Gets or sets the maximum allowed model state errors in this instance of <see cref="ModelStateDictionary"/>.
+        /// Defaults to <c>200</c>.
         /// </summary>
         /// <remarks>
-        /// The value of this property is used to track the total number of calls to
-        /// <see cref="AddModelError(string, Exception)"/> and <see cref="AddModelError(string, string)"/> after which
-        /// an error is thrown for further invocations. Errors added via modifying <see cref="ModelState"/> do not
-        /// count towards this limit.
+        /// <see cref="ModelStateDictionary"/> tracks model errors added by calls to
```

Comment:
will change to `tracks the number of model errors`. We don't need to spill our guts about how the code is implemented. From the user's POV what they are doing is adding errors, so the language uses that as a point of departure.